pkg/volumereplica/volumereplica.go

```diff
@@ -126,36 +128,36 @@ func PoolNameFromCVR(cvr *cstor.CStorVolumeReplica) string {
 
 // ToDo: Move this to backup package
 
-//// PoolNameFromBackup gets the name of cstorpool from cstorvolumereplica label
-//// if not found then gets cstorpoolinstance name from the OPENEBS_IO_POOL_NAME
-//// env
-//func PoolNameFromBackup(bkp *cstor.CStorBackup) string {
-//	poolname := bkp.Labels[CStorPoolUIDKey]
-//	if strings.TrimSpace(poolname) == "" {
-//		poolname = os.Getenv(string("OPENEBS_IO_POOL_NAME"))
-//		if strings.TrimSpace(poolname) == "" {
-//			return ""
-//		}
-//	}
-//	return PoolPrefix + poolname
-//
-//}
+// PoolNameFromBackup gets the name of cstorpool from cstorvolumereplica label
+// if not found then gets cstorpoolinstance name from the OPENEBS_IO_POOL_NAME
+// env
+func PoolNameFromBackup(bkp *openebsio.CStorBackup) string {
```

Comment:
nit: we will be having only one pool, so we can remove this function, fetching pool name from backup/restore resource.